conf/distro/rpb-qt5.conf

```diff
@@ -0,0 +1,8 @@
+require conf/distro/include/rpb.inc
+
+DISTRO_NAME = "Reference-Platform-Build-QT5"
+
+PACKAGECONFIG_append_pn-qtbase = " gles2 eglfs fontconfig kms"
```

Comment:
i tried to build the qt5 image with RPB/X11 distro and it failed. I think this is because gles2 and fontconfig should be enabled in rpb.inc, and we should keep eglfs and kms only here.  i will test that..